src/utility.cpp

```diff
@@ -10,6 +10,19 @@ int nearest_power_of_2(int num) {
     return round(exp2(round(log2(num))));
 }
 
+ImVec2 compute_max_target_size(const ImVec2 &target_size, const ImVec2 &box_size) {

```

Comment:
The declaration of `ImVec2` used here comes from `utility.cpp`.
Although it works, it is better to `#include <imgui.h>` to get `ImVec2`.

Reply:
Resolved here. https://github.com/shangchiwu/advanced-image-processor/pull/41/files#diff-02a2971fb3961e26e4d94fdc7b4dd2d48e887971f46d0f64127e5bef20814799R8